src/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+  list-style-type: none;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+body > div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-shadow: 2px 2px 3px 2px rgb(171, 163, 163);
+  margin: 30px;
+  width: 300px;
+}
```

Comment:
Your code is very clean and well organized. From what I can see, you have followed Javascript best practices through and the code is readable and concise. 
My only suggestion is an optional one and would not apply in all cases. However, your 'body' and 'body > div' have similar properties. You can apply DRY by combining the similar properties and applying to both selectors simultaneously. Then have only the unique properties for each selector.